.github/workflows/e2e.yml

```diff
@@ -0,0 +1,130 @@
+name: e2e
+
+on:
+  push:
+  pull_request:
+    branches: [ master, e2e-testing ]
+env:
+  REGISTRY: ${{ secrets.REGISTRY}}
+  go_version: '1.16.8'
+  golangci_version: '1.37.0'
+jobs:
+  e2e:
+    runs-on: ubuntu-20.04
+    if: "contains(github.event.head_commit.message, '[e2e]')"
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+      
+    - name: add operators dir
+      run: |
+        cd ../
+        git clone https://github.com/dell/csi-baremetal-operator.git 
+
+    - name: Set environment variables
+      run: |
+        echo "CSI_BAREMETAL_DIR=$RUNNER_WORKSPACE/csi-baremetal/" >> $GITHUB_ENV
+        echo "CSI_BAREMETAL_OPERATOR_DIR=$RUNNER_WORKSPACE/csi-baremetal-operator/" >> $GITHUB_ENV
+
+    - name: LVM2 install
+      run: sudo apt-get install -y lvm2
+
+    - name: Prepare Golang 
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{env.go_version}}
+
+    - name: Install helm
+      run: |
+        curl https://baltocdn.com/helm/signing.asc | sudo apt-key add -
+        sudo apt-get install apt-transport-https --yes
+        echo "deb https://baltocdn.com/helm/stable/debian/ all main" | sudo tee /etc/apt/sources.list.d/helm-stable-debian.list
+        sudo apt-get update
+        sudo apt-get install helm
+
+    - name: CUSTOM KIND
+      run: |
+        wget -O kind https://github.com/painhardcore/builder/releases/download/0.0.1-alpha/kind_v0.8.1
+        sudo chmod +x kind
+        sudo mv kind /usr/bin
+          
+    - name: Kubectl install
+      run: |
+        curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl
+        chmod +x ./kubectl
+        sudo mv ./kubectl /usr/local/bin/kubectl
+        kubectl version --client 
+
+    - name: add CSI_VERSION
+      run: |
+        cd ${{env.CSI_BAREMETAL_DIR}}
+        echo "CSI_VERSION=`make version`" >> $GITHUB_ENV
+
+    - name: Get dependencies
+      run: make dependency
+
+    - name: Compile proto files
+      run: |
+        sudo apt install protobuf-compiler
+        sudo apt install golang-goprotobuf-dev
```

Comment:
When I execute `make install-compile-proto` it can’t found path to `protoc-gen-go`, which you install by `go get` in last step of `install-proto`. But when I try to add `protoc-gen-go` manually to path, execution file doesn’t exist in workflow. Therefore, I add libraries that have all the necessary files.

Reply:
Got it. You can remove this step and add TODO comment here to use custom path for protoc (and create a github issue as well). 
It's not good to use duplicate libraries. We can't be sure about its version and content